app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for lt in phrase:
+        if lt.lower() == letter.lower():
+            counter += 1
```

Comment:
Yeah, this solution works, but I recommend you to use count() method of string. You'll get a beautiful 1 line solution ;)